tests/test_authentication.py

```diff
@@ -126,28 +127,11 @@ def test_2fa_required(self) -> None:
     def test_successful_token_validation(self) -> None:
         base_dir = os.path.join(self.fixtures_path, inspect.stack()[0][3])
         cookie_dir = os.path.join(base_dir, "cookie")
+        cookie_master_path = os.path.join(self.root_path, "cookie")
 
-        for dir in [base_dir, cookie_dir]:
-            recreate_path(dir)
+        recreate_path(base_dir)
 
-        # We need to create a session file first before we test the auth token validation
-        with vcr.use_cassette(os.path.join(self.vcr_path, "2sa_flow_valid_code.yml")):
-            runner = CliRunner(env={"CLIENT_ID": "DE309E26-942E-11E8-92F5-14109FE0B321"})
-            result = runner.invoke(
-                main,
-                [
-                    "--username",
-                    "jdoe@gmail.com",
-                    "--password",
-                    "password1",
-                    "--no-progress-bar",
-                    "--cookie-directory",
-                    cookie_dir,
-                    "--auth-only",
-                ],
-                input="0\n654321\n",
-            )
-            assert result.exit_code == 0
+        shutil.copytree(cookie_master_path, cookie_dir)
```

Comment:
Does this change have to be done in the following files as well?
* [test_listing_libraries.py](https://github.com/icloud-photos-downloader/icloud_photos_downloader/blob/8b3adb04eb13cf47c5684530348c7ee347c85fcc/tests/test_listing_libraries.py#L25-L28)
* [test_listing_albums](https://github.com/icloud-photos-downloader/icloud_photos_downloader/blob/8b3adb04eb13cf47c5684530348c7ee347c85fcc/tests/test_listing_albums.py#L26-L30)

Reply:
Per [comment](https://github.com/icloud-photos-downloader/icloud_photos_downloader/issues/970#issuecomment-2436796863), this is just a missed change in [PR 971](https://github.com/icloud-photos-downloader/icloud_photos_downloader/pull/971)

Reply:
Yes, @AndreyNikiforov mentioned that he will fix it. I will then rebase this branch and the tests should pass.

Reply:
Rebased. Also fixed CodeQL and type check warnings. Now this PR is ready.